src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
-}
+  const result = {};
+  const sourceArray = sourceString.split(';');
+
+  sourceArray.map(elem => {
```

Comment:
You are using map function but not returning anything from it. The map function should be used when you want to transform elements in an array. In this case, you want to perform side effects (modifying external 'result' variable), so 'forEach' is more appropriate.

Reply:
fixed